cogs/channel.py

```diff
@@ -0,0 +1,144 @@
+import os
+import sys
+import csv
+import yaml
+from discord.ext import commands
+from discord.ext.commands import has_permissions
+from discord.utils import find
+import re
+
+if "CompsciBot" not in str(os.getcwd()):
+    os.chdir("./CompsciBot")
+with open("config.yaml") as file:
+    config = yaml.load(file, Loader=yaml.FullLoader)
+
+# Here we name the cog and create a new class for the cog.
+testGuild = 931309621027684413
```

Comment:
Is this intended to be in the final code?

Reply:
This is for a different feature that I accidentally committed.  I will remove it.